### PR TITLE
Sidebar bugs

### LINKED ISSUE #144: Sidebar bugs
Fix the bugs with the typeahead and the buy button in the sidebar.

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Fix bug in sidebar buy button. 

Closes #144

### CODE REVIEW

File: pokeapp/src/components/PokemonCard.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ const PokemonCard = ({
           >
             Buy
           </div>
-          {pokemon.quantity > 0 ? (
+          {pokemon.quantity > 0 && active ?  (
```
Comment (krferrell) [inline pokeapp/src/components/PokemonCard.js:109]: Nice catch on this one.

File: pokeapp/src/components/PokemonCard.js [hunk 2]
```diff
@@ -122,8 +122,10 @@ const PokemonCard = ({
       </div>
 
       <div className="type-id-container">
-        <div className="pokemon-quantity">
-          {pokemon.quantity > 0 ? <p>x{pokemon.quantity}</p> : ""}
+        <div className={pokemon.quantity > 0 ? "pokeball" : "pokeball-hidden"}>
+          <div className="pokeball-top"></div>
+          <div className="pokeball-bottom"></div>
+          <h4>{pokemon.quantity > 0 ? `x${pokemon.quantity}` : ""}</h4>
```
Comment (krferrell) [inline pokeapp/src/components/PokemonCard.js:128]: Great work on this pokeball

Comment (gonzalespaulb) [inline pokeapp/src/components/PokemonCard.js:128]: Solid work rendering the pokeballs conditionally.

**Review by connielion — APPROVED** [review]
Nice ternary!

**Review by krferrell — APPROVED** [review]
Nice Jay!

File: pokeapp/src/styling/colorschemes/_colors.scss [hunk 3]
```diff
@@ -5,6 +5,7 @@ $component-container: hsl(0, 0%, 20%);
 
 $card-background: #0000004b;
 $card: #262626;
+$quantity-color: #000;
```
Comment (BlackCubes) [inline pokeapp/src/styling/colorschemes/_colors.scss:8]: Beautiful

**Review by BlackCubes — APPROVED** [review]
Bruh, dis a good bug catch

**Review by gonzalespaulb — APPROVED** [review]

**Review by JorgeLVilla — APPROVED** [review]
Nice work using conditionals Jay!

**Pull request merged**